src/chainstate/stacks/db/blocks.rs

```diff
@@ -5481,6 +5501,10 @@ impl StacksChainState {
                     cur_epoch.start_height,
                 )
             }
+
+            StacksEpochId::Epoch30 => {
+                todo!() // TODO(sbtc): Extend the logic of this function to return sbtc bitcoin-ops
```

Comment:
Please do this in this PR.  This PR should enable `peg_in` to be processed as part of the block, even if for now it's a no-op (i.e. since there's no smart contract state to mutate).

Reply:
Sure, I'll break out this todo to an epoch-3.0 branch independent of the PegIn changes.